### ISSUE TITLE
About the model zoo——using a small batchsize, but getting a much better result!!!

### ISSUE THREAD

**Issue by LaoYang1994** [facebookresearch/maskrcnn-benchmark#552]
## ❓ Questions and Help
I only have three GPUs, so I set the IMS_PER_BATCH to 6 (2* 3), and set the initial learning rate to 0.01( 0.02 * 3 / 8 = 0.0075, I think 0.0075 is a little low). To my surprise, I got a bbox AP of 39+, close to the reported AP of R_101_FPN.
I am thinking whether the Detectron learning rate setting is appropriate. @fmassa

**Comment by estimoelmo** [comment]
Is there any reason that you divided it by 8? I think, times 3 is good but not sure if we have 3 gpus as the [Detectron](https://github.com/facebookresearch/Detectron/blob/master/configs/getting_started/tutorial_2gpu_e2e_faster_rcnn_R-50-FPN.yaml) has the increment of 2

**Comment by LaoYang1994** [comment]
> Is there any reason that you divided it by 8? I think, times 3 is good but not sure if we have 3 gpus as the [Detectron](https://github.com/facebookresearch/Detectron/blob/master/configs/getting_started/tutorial_2gpu_e2e_faster_rcnn_R-50-FPN.yaml) has the increment of 2

Excuse me, could you please describe a little more detail? I didn't quite understand what you mean.

I adjust the learning according the batchsize. Because of the smaller batchsize, I reduce the learning rate to 0.01.

Waiting for your reply.

**Comment by zimenglan-sysu-512** [comment]
hi @LaoYang1994 
u should set `0.0075` instead of `0.00075`

**Comment by LaoYang1994** [comment]
hi @zimenglan-sysu-512 
I'm sorry, it's an oversight on my part. 
But the question is that why I get a better result when I use a small batchsize and a corresponding learning rate.
Waiting for your reply.

**Comment by fmassa** [comment]
@LaoYang1994 is this for COCO? Can you share the exact config / training arguments that you used to train your model?

**Comment by LaoYang1994** [comment]
Maybe I have found the reason why I got a better performance. My annotations of the coco dataset has been changed. I think it's the reason. I'm sorry for asking such a silly problem.
